src/Akka.Streams.Kafka/Stages/Consumers/Actors/KafkaConsumerActorMetadata.cs

```diff
@@ -73,18 +74,25 @@ public sealed record Subscribe(IImmutableSet<string> Topics, IPartitionEventHand
             public sealed record SubscribePattern(string TopicPattern, IPartitionEventHandler RebalanceHandler)
                 : ISubscriptionRequest;
 
+            /// <summary>
+            /// Marker interface for shutdown messages to the KafkaConsumerActor
+            /// </summary>
+            public interface IStopLike : INoSerializationVerificationNeeded;
```

Comment:
Used to help trace the ways in which the `KafkaConsumerActor` might be told to shut down